CHANGELOG.md

```diff
@@ -9,6 +9,7 @@ Changelog
 
 ### Fixes
 
+- Disabled button on forms when there is nothing to submit ([PR 1998](https://github.com/input-output-hk/daedalus/pull/1998))
```

Comment:
@DeeJayElly please move this line under `vNext` (add it if it is not already there!)

Reply:
Added